data/dr19.cfg

```diff
@@ -13,20 +13,32 @@ release_date = 2025-07-11
 #
 #
 [DATA]
+CAS_LOAD = %(DATA_ROOT)s/cas-load
+APOGEE_DATA_N = %(DATA_ROOT)s/apogee/apo
 FCAM_DATA_N = %(DATA_ROOT)s/fcam/apo
 GCAM_DATA_N = %(DATA_ROOT)s/gcam/apo
 #
 #
 #
 [SPECTRO]
+ALLSPEC = %(SPECTRO_ROOT)s/allspec
 APOGEE_REDUX = %(SPECTRO_ROOT)s/apogee/redux
-MWM_ASTRA= %(SPECTRO_ROOT)s/astra
+LVM_SPECTRO_REDUX = %(SPECTRO_ROOT)s/lvm/redux
```

Comment:
Should this go under a new `[LVM]` section, similar to the `[BHM]` and `[MWM]` sections?  An example of this is in `sdsswork.cfg`.

Reply:
As in my comment above, there are no top level bhm or mwm or lvm folders (unlike sdsswork), since as of DR18, we now have a top level spectro folder instead, so all of the spectro data products go under [SPECTRO] so the cfg file is as it should be!